http3/client.go

```diff
@@ -196,9 +196,12 @@ func (c *ClientConn) roundTrip(req *http.Request) (*http.Response, error) {
 		// wait for the handshake to complete
 		earlyConn, ok := c.Connection.(quic.EarlyConnection)
 		if ok {
+			trace := httptrace.ContextClientTrace(req.Context())
 			select {
 			case <-earlyConn.HandshakeComplete():
+				traceTLSHandshakeDone(trace, earlyConn.ConnectionState().TLS, nil)
```

Comment:
This is not correct. Not every request executes the TLS handshakes (namely, if a connection is reused).